requirements_dev.txt

```diff
@@ -1,9 +1,9 @@
 -r requirements.txt
-black==19.10b0
+black==20.8b1
```

Comment:
Make sure you update the `rev` for the black pre-commit hook to match, in .pre-commit-config.yaml. And then you'll need to run black on everything to pass the pre-commit hook.